config/sites/oniowa.uiowa.edu/config_split.patch.views.view.events.yml

```
@@ -15,6 +19,125 @@ adding:
+              target: ''
+              nl2br: false
+              max_length: 0
+              word_boundary: true
+              ellipsis: true
+              more_link: false
+              more_link_text: ''
+              more_link_path: ''
+              strip_tags: false
+              trim: false
+              preserve_tags: ''
+              html: false
+            element_type: ''
+            element_class: ''
+            element_label_type: ''
+            element_label_class: ''
+            element_label_colon: false
+            element_wrapper_type: ''
+            element_wrapper_class: ''
+            element_default_classes: true
+            empty: ''
+            hide_empty: false
+            empty_zero: false
+            hide_alter_empty: true
+            click_sort_column: value
+            type: boolean
+            settings:
+              format: boolean
+              format_custom_false: ''
+              format_custom_true: ''
+            group_column: value
+            group_columns: {  }
+            group_rows: true
+            delta_limit: 0
+            delta_offset: 0
+            delta_reversed: false
+            delta_first_last: false
+            multi_type: separator
+            separator: ', '
+            field_api_classes: false
+          title:
+            label: Title
+          field_event_when_end_value:
+            id: field_event_when_end_value
+            table: node__field_event_when
+            field: field_event_when_end_value
+            relationship: none
+            group_type: group
+            admin_label: ''
+            plugin_id: date
+            label: ''
+            exclude: true
+            alter:
+              alter_text: false
+              text: ''
+              make_link: false
+              path: ''
+              absolute: false
+              external: false
+              replace_spaces: false
+              path_case: none
+              trim_whitespace: false
+              alt: ''
+              rel: ''
+              link_class: ''
+              prefix: ''
+              suffix: ''
+              target: ''
+              nl2br: false
+              max_length: 0
+              word_boundary: true
+              ellipsis: true
+              more_link: false
+              more_link_text: ''
+              more_link_path: ''
+              strip_tags: false
+              trim: false
+              preserve_tags: ''
+              html: false
+            element_type: ''
+            element_class: ''
+            element_label_type: ''
+            element_label_class: ''
+            element_label_colon: false
+            element_wrapper_type: ''
+            element_wrapper_class: ''
+            element_default_classes: true
+            empty: ''
+            hide_empty: false
+            empty_zero: false
+            hide_alter_empty: true
+            date_format: custom
+            custom_date_format: 'g:i A'
```

There's some leeway, but the UI editorial guide usually has lowercase

>Time: a.m. and p.m.
>The abbreviations “a.m.” and “p.m.” are lowercase with periods and no space between letters. There should
be a single space between “a.m.” or “p.m.” and the numeral that precedes them.
>6 a.m.
>10 p.m.
>In display type, tabular material, or lists, “a.m.”/“p.m.” style may vary depending on graphic design. For
example, small caps may be used, or the periods may be deleted. [UI]

```suggestion
            custom_date_format: 'g:i a'
```